src/assets/scss/modules/description.scss

```diff
@@ -1,81 +0,0 @@
-.description {
-  padding: 50px 0 100px;
-  @include backgroundImg("/assets/img/description.png");
```

Comment:
It is a good practice to include mixins before or after all css rules, but do not mix them. Here you could have added `@include` either before padding or after text-align.